### PR TITLE
move new features to introduction, fix missing images

### PR DESCRIPTION
**Pull request by carlaschroder**
Some images were not displaying because of incorrect alt text formatting.

### CODE REVIEW

**Comment by phil-davis** [comment]
client-6.png - my Version 2.0.2-nightly20150915 (build 5463) has a different quota usage bar "500MB of 1.5GB in use":
a202

File: doc/introduction.rst [hunk 1]
```diff
@@ -14,4 +14,43 @@ and local PC.
...
+  * Accessibility improvements (high contrast schemes)
+  * Automatic Bandwidth Throttling
+  * Don't show redundant directory entries in activity log
+  * Remove deleted accounts properly from toolbar
+  * File manager integration: Show hidden files as ignored
+  * Show wizard when last account was deleted
+  * Do not sync down new big folders from server without user's consent
+  * Integrate Selective Sync into the default UI
+  * More reliable reconnect after timeout
+  * Use SI units for the file sizes
```
Comment (phil-davis) [inline doc/introduction.rst]: What to say here? In 2.0.2 it goes back to showing units as KB, MB, GB but actually calculating underneath in 1024 multiples.

**Comment by carlaschroder** [comment]
Looks like I need all new screenshots.

**Comment by carlaschroder** [comment]
Allrightythen, this is my final answer & it is ready to merge.

**Comment by carlaschroder** [comment]
:+1:

**Comment by danimo** [comment]
@carlaschroder How much would you hate me if I'd ask you to do the screenshot on Windows (or Mac)? Since we have >70% Windows users it might be a good idea, plus Linux never looks the same (Exhibit A: the systray screenie of yours).

**Comment by carlaschroder** [comment]
@danimo Oh, a lot :) In the final edit of installing.rst I used only the first 4 screenshots. They came from my testpilot installation, so I hacked them to remove all the 'testpilot' wording. I think they look nice. How about I capture all future screenshots from my windows 7 machine.

**Comment by phil-davis** [comment]
I guess if you are not using client-6 then nobody will see that it is out-of-date. But someone might just use it in future assuming it is an up-to-date UI shot.
While you are at it you could take the free upgrade to Win10 on your Win7 machine :)

**Comment by carlaschroder** [comment]
> While you are at it you could take the free upgrade to Win10 on your Win7 machine
@phil-davis Ew nooo!!

**Comment by carlaschroder** [comment]
allrighty then @danimo, this is my realio trulio final answer! Nice new screenshots for client-1.png through client-4.png for the revised installation page. Just those four, and the ugly ones all gone. Please merge this so I can push the remaining updates.

**Comment by phil-davis** [comment]
:+1:

**Pull request merged**